app/[locale]/academics/departments/[name]/page.tsx

```diff
@@ -120,12 +299,53 @@ export default async function Department({
         id="programmes"
         text={text.headings.programmes.toUpperCase()}
       />
-      <nav className="container">{/* Programmes */}</nav>
+      <nav className="container mx-auto grid grid-cols-1 gap-y-7 lg:grid-cols-3 lg:gap-4">
+        <ProgrammesSectionItem
+          title={'UNDER GRADUATE'}
+          programmeType={'B. Tech'}
+          items={[
+            'B.tech Computer Engineering',
+            ' Information Technology',
+            'Artificial Intelligence / Machine Learning',
+          ]}
+          textColor={'text-primary-300'}
+          bgHoverColor={'bg-primary-300'}
+        />
+        <ProgrammesSectionItem
+          title={'POST GRADUATE'}
+          programmeType={'M. Tech'}
+          items={['M.tech Computer Engineering', ' Cyber Security']}
+          textColor={'text-primary-700'}
+          bgHoverColor={'bg-primary-700'}
+        />
+        <ProgrammesSectionItem
+          title={'DOCTORAL'}
+          programmeType={'Ph. D'}
+          items={['More Information']}
+          textColor={'text-primary-900'}
+          bgHoverColor={'bg-primary-900'}
+        />
+      </nav>
 
-      <nav className="container">
-        {/* Faculty & Staff */}
-        {/* Laboratories */}
-        {/* Student Achievements */}
+      <nav
+        className={cn(
+          'container',
+          'my-10 md:my-12 lg:my-16 xl:my-20',
+          'flex flex-col space-y-5 lg:flex-row lg:justify-between lg:space-y-0'
+        )}
+      >
+        {[
+          { icon: MdBadge, title: 'Faculty And More' },
+          { icon: ImLab, title: 'Laboratories' },
+          { icon: MdMilitaryTech, title: 'Student Achivements' },
+        ].map((obj, i) => (
+          <AchievementLink
+            key={i}
+            href={''}
+            icon={obj.icon}
+            title={obj.title}
+          />
```

Comment:
Why make a component for this? The component's return JSX is 8 LOC and it's not used anywhere else.